opengrok-indexer/src/main/java/org/opengrok/indexer/history/Repository.java

```diff
@@ -86,7 +86,7 @@ public abstract class Repository extends RepositoryInfo {
      * List of &lt;revision, tags&gt; pairs for repositories which display tags
      * only for files changed by the tagged commit.
      */
-    protected TreeSet<TagEntry> tagList = null;
+    protected NavigableSet<TagEntry> tagList = null;
```

Comment:
what is the navigable set used for ?

Reply:
NavigableSet is the interface implemented by TreeSet. It is recommended to use interfaces instead of implementation.

Reply:
I see.